lib/gui/lvgl/Kconfig.upstream

```diff
@@ -0,0 +1,867 @@
+# Kconfig file for LVGL v8.0
+
+menu "LVGL configuration"
+
+    # Define CONFIG_LV_CONF_SKIP so we can use LVGL
+    # without lv_conf.h file, the lv_conf_internal.h and
+    # lv_conf_kconfig.h files are used instead.
+    config LV_CONF_SKIP
+        bool
+        default y
+
+    config LV_CONF_MINIMAL
+        bool "LVGL minimal configuration."
```

Comment:
I know this file is from upstream, so in principle it shouldn't be reviewed here, but want to make a comment on this.

Looking at the code, this doesn't mean a minimal configuration which a user might expect.
Instead it means a different set of defaults which is not the same as a minimal configuration.

For example it suffers from the stuck symbol syndrome:
https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#stuck-symbols-in-menuconfig-and-guiconfig

Test for yourself, for example:
```
$ cmake -GNinja -DBOARD=mimxrt1050_evk -Bbuild samples/hello_world/ -DCONFIG_LVGL=y

# Open menuconfig:
$ ninja -Cbuild menuconfig

# Navigate to: `Additional libraries → LittlevGL GUI library → LVGL configuration`
# Enable '[*] LVGL minimal configuration.'
```
and then check that the value of `LV_USE_ASSERT_NULL` is still enabled despite the fact that Minimal configuration has just been enabled.
https://github.com/zephyrproject-rtos/zephyr/blob/f89724bd78b1c2c090398f8cc7ce4f393d4f8769/lib/gui/lvgl/Kconfig.upstream#L300-L302

Reply:
@kisvegabor FYI ^^

Reply:
Done: https://github.com/lvgl/lvgl/issues/2976